packages/core/src/global.js

```diff
@@ -72,7 +72,7 @@ export let Global: React.AbstractComponent<
     return (
       <style
         {...{
-          [`data-emotion-${cache.key}`]: serializedNames,
+          [`data-emotion-global`]: `${cache.key} ${serializedNames}`,
```

Comment:
had to differentiate this with `-global` so it doesnt get rehydrated into "main" sheet